lua/lundump.c

```diff
@@ -238,8 +238,8 @@ static void LoadHeader (LoadState* S)
 	"read version %d.%d; expected at least %d.%d",
 	S->name,V(version),V(VERSION0));
  S->swap=(luaU_endianness()!=LoadByte(S));	/* need to swap bytes? */
```

Comment:
Yeah, we should probably really change the thing where xoreos' C code (lua, glew) is build with a C compiler instead of a C++ one. That way, we can pull in the xoreos endian code. I've got a few things I want to change in the lua tree on my TODO list there.

I'd queue that for after this PR has been merged, though. I'd just introduce merge conflicts needlessly.